FAQ.md

```diff
@@ -0,0 +1,22 @@
+# Foire Aux Questions
+
+Cette page répertorie les questions fréquemment rencontrées au sujet de l'environnement technique d'OpenFisca.  
+N'hésitez pas à l'[enrichir](https://github.com/openfisca/tutorial/edit/master/FAQ.md) de vos expériences ! :slightly_smiling_face: 
+
+
+> Sous Windows, il est supposé que vous disposez d'un terminal bash (i.e [GitBash](https://gitforwindows.org) ou assimilé).
+
+## GIT
+
+| ERREUR | CONTEXTE | SOLUTION      |
+| ------ | -------- | ------------- |
+| `could not read from remote repository github` | Vous tentez de pousser votre travail en local sur le serveur avec : </br>`git push` | * S’assurer de l’existence d’une clef SSH avec `ls ~/.ssh/` </br>Attendu : une clef publique que nous nommerons `fichier_ssh.pub` et sa clef privée associée `fichier_ssh`. </br>* Le contenu de fichier_ssh.pub doit être référencé dans les settings du compte GitHub de l’usager. </br>* Ajouter la clef privée SSH à votre ssh-agent avec `ssh-add ~/.ssh/fichier_ssh` </br>Pour en savoir plus, voir l'[aide GitHub/SSH](https://help.github.com/articles/connecting-to-github-with-ssh/). |
+| `could not open a connection to your authentication agent` | Pour vous identifier auprès d'un service tel que GitHub, vous tentez d'exporter votre clef SSH en l'ajoutant au cache de votre agent SSH avec : </br>`ssh-add ~/.ssh/fichier_ssh` | * S’assurer de l’exécution du ssh agent avec `eval $(ssh-agent)` </br></br>Et/OU </br></br>* S’assurer que l’adresse du dépôt n'est pas en HTTPS car elle doit être en SSH : `git remote -v`. </br>Le résultat attendu pour le dépôt tutorial est : </br>`origin	git@github.com:openfisca/tutorial.git (fetch)` </br>`origin	git@github.com:openfisca/tutorial.git (push)` </br>Pour en savoir plus, voir l'[aide GitHub](https://help.github.com/articles/changing-a-remote-s-url/#switching-remote-urls-from-ssh-to-https). |
+
+## Python OpenFisca
+
+| ERREUR | CONTEXTE | SOLUTION      |
+| ------ | -------- | ------------- |
+| `command not found` | Vous tentez d'appeler l'interprète du langage Python avec : `python un_fichier.py` | Mettre à jour la variable d'environnement `PATH` : </br>* Vérifier sa valeur avec : </br>`echo $PATH` </br>* Puis, retrouvez le répertoire où la version souhaitée de Python a été installée ; celui-ci varie d'un environnement à l'autre. </br>Exemple Windows : `/c/Python27`. </br>Exemple Unix : `/opt/local/bin/python` </br>* Mettre à jour la variable en indiquant le chemin vers l'interprète python. </br>Exemple Windows : `export PATH=/c/Python27:$PATH`. </br>Exemple Unix : `export PATH=/opt/local/bin:$PATH` </br></br>Afin que cette mise à jour soit permanente, il est conseillé d'ajouter cette commande d'export à la fin de votre fichier de configuration Bash `~/.bashrc`. |
```

Comment:
for windows user, wouldn't it be interesting to show them how to do it in the visual interface ?

Reply:
Updating it in the visual interface wouldn't solve their problem in their GitBash.
This would be interesting later when we solve how to use OpenFisca in full MS-DOS/cmd.exe environment. 😍